017-quartz-cluster/src/test/java/io/quarkus/qe/quartz/AnnotationScheduledJobsQuartzTest.java

```diff
@@ -50,9 +53,8 @@ private void whenBothNodesAreUpAndRunning() {
         assertFalse(nodeTwoApp.getStartupConsoleOutput().isEmpty(), "Node Two should be up and running");
     }
 
-    private void whenShutdownNodeOne() throws Exception {
-        // TODO: Workaround for https://github.com/quarkusio/quarkus/issues/14961
-        nodeOneApp.afterAll(null);
+    private void whenShutdownNodeOne() {
+        nodeOneApp.stop();
```

Comment:
https://github.com/quarkusio/quarkus/issues/14961 is fixed for 1.13, not 1.11.

We still primarily care about 1.11 ...

Reply:
This change is already in another module in master.
I think the way to go is to disable these modules in master?

Reply:
ah, I guess you mean 018-quarkus-runtime-properties ++ 019-quarkus-consul ...

Let's keep it in master, adjustments need to be handled in 1.11 branch PR